R/SurveyMap.R

```diff
@@ -115,19 +115,19 @@
 #' )
 #'
 #' # Example lme4 usage
-#' # fit_2 <- ex_map$fit(
-#' #   fun = lme4::glmer,
-#' #   formula = y ~ (1|age) + (1|gender),
-#' #   family = "binomial"
-#' # )
-#' #
+#' fit_2 <- ex_map$fit(
+#'   fun = lme4::glmer,
+#'   formula = y ~ (1|age) + (1|gender),
+#'   family = "binomial"
+#' )
+#'
 #' # Example brms usage
-#' # fit_3 <- ex_map$fit(
-#' #   fun = brms::brm,
-#' #   formula = y ~ (1|age) + (1|gender),
-#' #   family = "bernoulli",
-#' #   seed = 1111
-#' # )
+#' fit_3 <- ex_map$fit(
+#'   fun = brms::brm,
+#'   formula = y ~ (1|age) + (1|gender),
+#'   family = "bernoulli",
+#'   seed = 1111
+#' )
```

Comment:
I think the brms example needs to be commented out so it doesn't run by default. I don't think we can assume the user has their compiler/rstan/cmdstanr configured and if they don't this will result in the example erroring. That also means that we need to either comment out any uses of `fit_3` below or not use `fit_3`.